.travis.yml

```
@@ -200,6 +196,21 @@ jobs:
           --env AWS_SECRET_ACCESS_KEY="$AWS_SECRET_ACCESS_KEY"
           opendap/rocky9_hyrax_builder:latest /root/bes/travis/build-rh9-rpm.sh
 
+    - stage: build-and-package
+      name: "Build bes docker image (RHEL8)"
+      script:
+        - export BES_BUILD=docker-rhel8
+        - echo "branch name ${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}"
+        - export DOCKER_NAME="bes_rhel8"
+        - export DIST=el8
+        - export OS=rocky8
+        - export BES_REPO_DIR=${TRAVIS_BUILD_DIR}
+        - export SNAPSHOT_IMAGE_TAG="opendap/${DOCKER_NAME}:snapshot"
+        - export BUILD_VERSION_TAG="opendap/${DOCKER_NAME}:${BES_BUILD_NUMBER}"
```

Hrm. We probably want this to be 
```suggestion
        - export BUILD_VERSION_TAG="opendap/${DOCKER_NAME}:${BES_VERSION}-${BES_BUILD_NUMBER}"
```
...but the version only happens after the build, and we'd have to get it from within the dockerfile. Hrm. I'm going to have to think about this.

---

figured it out!